Doc/whatsnew/3.12.rst

```diff
@@ -237,6 +237,10 @@ asyncio
 * Add C implementation of :func:`asyncio.current_task` for 4x-6x speedup.
   (Contributed by Itamar Ostricher and Pranav Thulasiram Bhat in :gh:`100344`.)
 
+* :func:`asyncio.iscoroutine` now returns :const:`False` for generators as
```

Comment:
As per the devguide's recommendation: https://devguide.python.org/documentation/markup/#quick-reference

The reason why is that we assume that the vast majority of Python devs reading the docs will know what True, False and None refer to, so the link will just be distracting.

```suggestion
* :func:`asyncio.iscoroutine` now returns ``False`` for generators as
```

Reply:
Ok, thanks